static/app/views/starfish/queries/useMetrics.ts

```diff
@@ -18,13 +23,29 @@ interface UseSpanMetricsOptions<Fields> {
 }
 
 export const useSpanMetrics = <Fields extends SpanMetricsProperty[]>(
-  options: UseSpanMetricsOptions<Fields> = {}
+  options: UseMetricsOptions<Fields> = {}
+) => {
+  return useMetricsBase<Fields, SpanMetricsResponse>(
+    options,
+    DiscoverDatasets.SPANS_METRICS
+  );
+};
+
+export const useMetrics = <Fields extends MetricsProperty[]>(
+  options: UseMetricsOptions<Fields> = {}
+) => {
+  return useMetricsBase<Fields, MetricsResponse>(options, DiscoverDatasets.METRICS);
+};
+
+const useMetricsBase = <T extends Extract<keyof ResponseType, string>[], ResponseType>(
```

Comment:
The naming here is a bit akward, was going for `use<Dataset>`, but i think specifying `base` suffix is a good workaround